src/scenes/AcceleratorRouter/ParticipantProjects/Reports/NewProjectSpecificMetric.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { Container, Grid, Typography, useTheme } from '@mui/material';
+import { Dropdown } from '@terraware/web-components';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+import Page from 'src/components/Page';
+import Card from 'src/components/common/Card';
+import Checkbox from 'src/components/common/Checkbox';
+import PageForm from 'src/components/common/PageForm';
+import TextField from 'src/components/common/Textfield/Textfield';
+import { APP_PATHS } from 'src/constants';
+import { selectCreateProjectMetric } from 'src/redux/features/reports/reportsSelectors';
+import { requestCreateProjectMetric, requestProjectReportConfig } from 'src/redux/features/reports/reportsThunks';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import strings from 'src/strings';
+import { NewMetric } from 'src/types/AcceleratorReport';
+import useForm from 'src/utils/useForm';
+
+export default function NewProjectSpecificMetric(): JSX.Element {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  const pathParams = useParams<{ projectId: string }>();
+  const projectId = Number(pathParams.projectId);
+  const dispatch = useAppDispatch();
+  const [requestId, setRequestId] = useState<string>('');
+  const createProjectMetricResponse = useAppSelector(selectCreateProjectMetric(requestId));
+
+  useEffect(() => {
+    if (projectId) {
+      dispatch(requestProjectReportConfig(projectId));
+    }
+  }, [projectId]);
+
+  const { isMobile } = useDeviceInfo();
+
+  useEffect(() => {
+    if (createProjectMetricResponse?.status === 'success') {
+      goToProjectReports();
+    }
+  }, [createProjectMetricResponse]);
+
+  const goToProjectReports = () => {
+    navigate(`${APP_PATHS.ACCELERATOR_PROJECT_REPORTS.replace(':projectId', projectId.toString())}?tab=settings`);
+  };
+
+  const [newMetric, , onChange] = useForm<NewMetric>({
+    component: 'Biodiversity',
+    name: '',
+    reference: '',
+    type: 'Activity',
+  });
```

Comment:
I don't see `description` in this but I see we're using it later on in the PageForm itself. Is that something that needs added here or is it null by default and we're ok with that?

Reply:
It can be null, that's why I didn't initialize it with a value here